packages/terra-site/src/examples/card/CardDefault.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Card from 'terra-card';
+
+
+const CardDefault = () => (
+  <div>
+    <Card>Hello World!!</Card>
```

Comment:
passing `hasPaddingHorizontal`, `hasPaddingVertical` to the parent component would just result in them being passed as custom props to the div. https://github.com/cerner/terra-core/pull/727/files#diff-0e7039b4f3fffa1ed4b9e7d014fd1211R26
Do we want to facilitate passing the props to either Card or Card.Padding components where then most people will always use just Card, and Card.Padding is an escape hatch for people with a horizontal rule separating 2 pieces of content. 
Or just keep it as we have it and we will usually use the Card + Card.Padding ?